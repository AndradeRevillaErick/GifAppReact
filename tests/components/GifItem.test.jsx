import { render } from "@testing-library/react";
import { GifItem } from "../../src/components/GifItem";

describe('Pruebas en <GifItem/>', () => {

    test('must to match with snapshot', () => {

        // const props = {
        //     title: 'Este es el titulo',
        //     url: 'Esta es la URL'
        // }

        // const { container } = render( <GifItem { ...props }/>);

        const title = 'Dragon ball';
        const url   = 'https://dragon-ball.com/goku.jpg';

        const { container } = render( <GifItem title={ title } url={ url }/>);

        expect( container ).toMatchSnapshot();
    });
})