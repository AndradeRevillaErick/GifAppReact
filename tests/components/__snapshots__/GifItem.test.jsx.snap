// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifItem/> must to match with snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Dragon ball"
      src="https://dragon-ball.com/goku.jpg"
    />
    <p>
      Dragon ball
    </p>
  </div>
</div>
`;
